src/index.css

```diff
@@ -0,0 +1,114 @@
+.container {
+  width: 30%;
+  border: solid 2px;
+  max-height: 450px;
+  margin: 125px auto;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.add-to-do {
+  position: relative;
+  height: 40px;
+  background-color: #fff;
+  padding: 10px;
+  border-top: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.add-to-do input {
+  width: 100%;
+  background-color: transparent;
+  border: none;
+  font-size: 20px;
+  padding-left: 10px;
+}
+
+.content {
+  background-color: #fff;
+}
+
+#task {
+  display: flex;
+  list-style: none;
+  padding: 0 0 0 20px;
+  justify-content: space-between;
+  height: 60px;
+}
+
+#task p {
+  margin: 0;
+  font-size: 20px;
+  padding-right: 200px;
+}
+
+#task .checked {
+  text-decoration: line-through;
+}
+
+.task-menu {
+  margin: 10px 0 0 0;
+  padding: 0;
+  list-style: none;
+}
+
+#task .setting {
+  padding-right: 20px;
+  cursor: pointer;
+  position: relative;
+}
+
+.setting .task-menu {
+  position: absolute;
+  padding: 5px 0;
+  left: -80px;
+  bottom: 20px;
+  transform: scale(0);
+  background: #fff;
+  z-index: 2;
+  border-radius: 5px;
+  transform-origin: top right;
+  transition: transform 0.3s ease;
+}
+
+/* .setting:hover .task-menu {
+  transform: scale(1);
+} */
```

Comment:
-  [ ] [OPTIONAL] Kindly remove all unused code in order to have a clean code-base.